### DISCUSSION TITLE
Rollback to a previous revision

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Revisions#953] [Feature Requests]
> My client has a use-case that I'm wondering if one of your plugins would be able to fulfill. Basically, they have lots of seasonal changes they need to make on pages that they need reverted back at a certain time.
> 
> It would go something like this:
> 1. Page's original content is made
> 2. Changes are made to page that are meant to be live for a month
> 3. Page reverts back to original content after one month automatically

**Comment by KratosGemini** [comment]
@stevejburge, an idea for you if it helps. We accomplish this by making 2 revisions of the original content at the time of editing. One revision we change and set up for release. The other revision we leave as is and schedule it for whenever the content needs to revert back.

This can mean a lot of revisions in the queue, but we don't have a lot of content where this is needed, so it hasn't been a problem.

**Comment by stevejburge** [comment]
Thanks @KratosGemini
